composer.json

```diff
@@ -173,6 +173,12 @@
         "box-compile": [
             "php build/box.phar compile"
         ],
+        "infection-install": [
+            "curl -f -L https://github.com/infection/infection/releases/download/0.26.0/infection.phar -o build/infection.phar"
```

Comment:
Why use a phar file rather than a composer require?

Reply:
They [don't recommend installing locally via Composer](https://infection.github.io/guide/installation.html). I'm not sure why, and I didn't feel like investigating 😄